tags/models.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+
+from associations.models import (
+    File,
+    Product,
+    Loanable,
+    Event,
+    Page,
+    Choice,
+    Role,
+    Folder,
+    Loan,
+    Association,
+)
+from forum.models import Theme, Topic, MessageForum
+
+
+class Namespace(models.Model):
+    class Meta:
+        unique_together = (("name", "scope", "scoped_to"),)
```

Comment:
See https://docs.djangoproject.com/en/2.2/ref/models/options/#unique-together, `unique_together` can be reduced to a single tuple.